### PR TITLE
Angolo attacco effettivo

### PR DESCRIPTION
**Pull request by MicheleMassa1**

### CODE REVIEW

File: Angolo_Attacco_Effettivo.m [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+%                < theta > blade pitch.
+%                < alpha_stall_up > maximum blade section angle of attack.
+%                < alpha_stall_lo > minimum blade section angle of attack.
+% |Output      : blade section angle of attack.
+% |Note        : u_R was calculated to be thorough; stall angles are
+%                imposed in order to visualize only the non-stalled region
+%                of the rotor.
+% ==============================================================================================
+
+function [alpha_e] = Angolo_Attacco_Effettivo(lambda,r_segn,beta,dbeta,mu,psi,theta,alpha_stall_up,alpha_stall_lo)
```
Comment (EttoreSaetta) [inline Angolo_Attacco_Effettivo.m]: English name needed.

**Review by EttoreSaetta — COMMENTED** [review]
You need to add a test case and a documentation.
After that we can review everything.

File: Angolo_Attacco_Effettivo.m (hunk 1 above)
Comment (MicheleMassa1) [reply Angolo_Attacco_Effettivo.m]: We added the test case and the documentation. We also changed the name of the function.

**Comment by EttoreSaetta** [comment]
@MicheleMassa1 the test case should be in the `TestCases` folder.

**Comment by MicheleMassa1** [comment]
@EttoreSaetta We created the TestCase folder.

**Comment by SerenaMang** [comment]
We applied the modifications required

**Comment by SerenaMang** [comment]
@EttoreSaetta

**Comment by EttoreSaetta** [comment]
Your test case should be in a sub-folder of the `TestCases` folder.

**Comment by SerenaMang** [comment]
@EttoreSaetta We created the sub-folder TestCase.

**Comment by EttoreSaetta** [comment]
> @EttoreSaetta We created the sub-folder TestCase.

@SerenaMang 
Your sub-folder name should be something that is referred to your function, `TestCase` is generic.
Into your test case sub-folder you have to remove the library function and leave only the test case script.

**Comment by SerenaMang** [comment]
@EttoreSaetta We renamed the sub-folder and removed the function from TestCases folder.

**Review by EttoreSaetta — APPROVED** [review]
@SerenaMang 
Approved.

**Pull request merged**